doc/tutorials/transformer/text_generation.ipynb

```diff
@@ -0,0 +1,224 @@
+{
```

Comment:
<p>please see my comment in the filling mask notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>